package.json

```
@@ -1,15 +1,17 @@
 {
   "name": "unicode-default-word-boundary",
-  "version": "13.0.0",
+  "version": "15.1.0",
   "description": "Implements the Unicode default word boundary specification (UAX #29 §4.1)",
   "main": "lib/index.js",
   "files": [
     "lib/**/*.js",
     "lib/**/*.d.ts"
   ],
   "scripts": {
-    "test": "tsc && ava",
-    "build": "./libexec/compile-word-break.js && tsc",
+    "test": "tsc && ava --verbose --timeout 5m",
+    "test-performance": "tsc && ava --verbose --timeout 5m --config ava-performance.config.cjs",
+    "ava": "ava --verbose --timeout 5m",
+    "build": "node ./libexec/compile-word-break.js && tsc",
```

`node` is unnecessary here:

```suggestion
    "build": "./libexec/compile-word-break.js && tsc",
```